Source/Extensions/Blazorise.Cropper/Cropper.razor.cs

```diff
@@ -324,6 +330,12 @@ internal async Task NotifyImageReady()
     /// This event fires when the image is ready / loaded.
     /// </summary>
     [Parameter] public Func<Task> ImageReady { get; set; }
+    
+    /// <summary>
+    /// This event fires when the image cannot be loaded. Usually because of 404 or <see cref="Source"/> being null.
+    /// Error message inside the string parameter.
+    /// </summary>
```

Comment:
We have a strict policy of not adding, or changing any APIs in the support channel. So I would move this event, `ImageLoadingFailed `,  to the next major milestone.

Reply:
Ok. I relaying on the addition not being a breaking change.

It's removed, I will put it back once this pr is synced with `master`.